bazel/coverage/fuzz_coverage_wrapper.sh

```diff
@@ -9,11 +9,11 @@ shift
 rm -rf fuzz_corpus
 
 mkdir -p fuzz_corpus/seed_corpus
-cp -r $@ fuzz_corpus/seed_corpus
+cp -r "$@" fuzz_corpus/seed_corpus
 
 # TODO(asraa): When fuzz targets are stable, remove error suppression and run coverage while fuzzing.
-LLVM_PROFILE_FILE= ${TEST_BINARY} fuzz_corpus -seed=${FUZZ_CORPUS_SEED:-1} -max_total_time=${FUZZ_CORPUS_TIME:-60} -max_len=2048 -rss_limit_mb=8192 -timeout=30 || true
+LLVM_PROFILE_FILE=${TEST_BINARY} fuzz_corpus -seed="${FUZZ_CORPUS_SEED:-1}" -max_total_time="${FUZZ_CORPUS_TIME:-60}" -max_len=2048 -rss_limit_mb=8192 -timeout=30 || :
 
 # Passing files instead of a directory will run fuzzing as a regression test.
-# TODO(asraa): Remove manual `|| true`, but this shouldn't be necessary. 
-${TEST_BINARY} $(find fuzz_corpus -type f) -rss_limit_mb=8192 || true
+# TODO(asraa): Remove manual `|| :`, but this shouldn't be necessary.
+${TEST_BINARY} "$(find fuzz_corpus -type f)" -rss_limit_mb=8192 || :
```

Comment:
im wondering if this will handle multiline output from `find` correctly

Reply:
I don't think it does:
```
$ printf "1: %s, 2: %s\n" $(printf "a\nb\n")
1: a, 2: b
$ printf "1: %s, 2: %s\n" "$(printf "a\nb\n")"
1: a
b, 2: 
```

Reply:
which without `readarray` means this i think
```
$ _args="$(printf "a\nb\n")"
$ while read -r line; do args+=($line); done \
    <<< "$_args"
$ printf "1: %s, 2: %s\n" "${args[@]}"
1: a, 2: b
```